ipn/ipnlocal/local.go

```diff
@@ -392,6 +392,23 @@ type LocalBackend struct {
 	//
 	// See tailscale/corp#29969.
 	overrideExitNodePolicy bool
+
+	// whether backend should use a hardware-backed key to bind the node
```

Comment:
// hardwareAttested is whether ...

https://go.dev/wiki/CodeReviewComments#comment-sentences